website/content/docs/k8s/multiport/configure.mdx

```diff
@@ -0,0 +1,478 @@
+
+<CodeBlockConfig filename="web.yaml">
+
+```yaml
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: web
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: web
+spec:
+  selector:
+    app: web
+  ports:
+    - port: 80
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: web
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: web
+  template:
+    metadata:
+      name: web
+      labels:
+        app: web
+      annotations:
+        "consul.hashicorp.com/mesh-inject": "true"
+        'consul.hashicorp.com/transparent-proxy': 'true'
+    spec:
+      containers:
+        - name: static-client
+          image: curlimages/curl:latest
+          # Just spin & wait forever, we'll use `kubectl exec` to demo
+          command: ['/bin/sh', '-c', '--']
+          args: ['while true; do sleep 30; done;']
+      serviceAccountName: web
+```
+
+</CodeBlockConfig>
+
+To apply these services to your Kubernetes deployment and register them with Consul, run the following command:
+
+```shell-session
+$ kubectl apply -f api.yaml -f web.yaml
+```
+
+## Configure traffic permissions
+
+Consul uses traffic permissions to validate communication between services based on L4 identity. In the beta release of the v2 catalog API, traffic permissions allow all services by default. In order to verify that services function correctly on each port, create CRDs that deny traffic to each port.
+
+The following examples create Consul CRDs that allow traffic to only one port of the multi-port service. Each resource separately denies `web` permission when it is a source of traffic to one of the services. These traffic permissions work with either method for defining a multi-port service.
+
+<CodeTabs tabs={[ "Deny port 80", "Deny port 90" ]}>
+
+<CodeBlockConfig filename="deny-80.yaml">
+
+```yaml
+apiVersion: auth.consul.hashicorp.com/v2beta1
+kind: TrafficPermissions
+metadata:
+  name: web-to-api-admin-port-allow
+spec:
+  destination:
+    identityName: api
+  action: deny
+  permissions:
+    - sources:
+        - identityName: web
+      destinationRules:
+        - portNames: ["api"]
+```
+
+</CodeBlockConfig>
+
+<CodeBlockConfig filename="deny-90.yaml">
+
+```yaml
+apiVersion: auth.consul.hashicorp.com/v2beta1
+kind: TrafficPermissions
+metadata:
+  name: web-to-api-port-allow
+spec:
+  destination:
+    identityName: api
+  action: deny
+  permissions:
+    - sources:
+        - identityName: web
+      destinationRules:
+        - portNames: ["admin"] 
+```
+
+</CodeBlockConfig>
+</CodeTabs>
+
+## Validate multi-port connection
+
+To open a shell to the `web` container, you need the name of the pod it currently runs on. Run the following command to return a list of pods:
+
+```shell-session
+$ kubectl get pods
+NAME                   READY   STATUS    RESTARTS   AGE
+api-7dc9d84f-kfnwz     2/2     Running   0          23s
+web-7bc5786747-b5pzl   1/1     Running   0          23s
+```
+
+### Validate both ports
```

Comment:
Curious as to why you didn't use an ordered list for each of these procedures. This one in particular seems well-suited for it.